hyperlane/testing/src/mock_validator_set.rs

```diff
@@ -117,6 +122,13 @@ impl MockValidatorSet {
         (message_id, raw_message, raw_metadata)
     }
 
+    pub fn validator_addresses(&self) -> BTreeSet<HexByteArray<20>> {
+        self.validators
+            .iter()
+            .map(|sk| HexByteArray::from_inner(derive_address(sk.verifying_key())))
```

Comment:
@Rhaki Let's call the function as `eth_utils::derive_address` instead of just `derive_address`. This makes it clear we're deriving an Eth address, not a Grug address.

```suggestion
            .map(|sk| HexByteArray::from_inner(eth_utils::derive_address(sk.verifying_key())))
```